osmosis/assetlist.json

```diff
@@ -1694,6 +1694,49 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/RAPTR.png"
         }
       ] 
+    },
+    {
+      "description": "Spitz on Osmosis",
+      "denom_units": [
+        {
+          "denom": "ibc/1570B676D4F35BACD92F470C01E4FD7B19CA7B8D62DA974A76A59079CEE338E3",
+          "exponent": 0,
+          "aliases": [
+            "spitz"
```

Comment:
please remove alias 'spitz' from denom for exponent 0, since it was used as the denom with exponent 6 above.